messages.rb

```diff
@@ -276,6 +289,92 @@ def send_list(event)
   event.attach_file(File::open(tmpfile))
 end
 
+def send_community(event)
+  msg = event.content
+  tabs = parse_tabs(msg)
+  tabs.empty? ? ftabs = "" : ftabs = format_tabs(tabs)
+  tabs = (tabs.empty? ? [:SI, :S, :SL, :SU, :SS, :SS2] : tabs)
+
+  levels = Score.where(rank: 0, highscoreable_type: "Level").includes(:level).where(levels: {tab: tabs})
+  episodes = Score.where(rank: 0, highscoreable_type: "Episode").includes(:episode).where(episodes: {tab: tabs})
+  lvls = levels.count
```

Comment:
I find these names a bit confusing since `levels` and `lvls` are basically the same term, same with `episodes` and `eps`. Maybe `n_levels` instead?

Reply:
True, I've changed all variable names to make them clearer. Also, since I've revamped this method and it now depends on a static Score class method, there's less trouble with variable names now.